apps/web-mzima-client/src/app/shared/components/company-info/company-info.component.scss

```diff
@@ -113,6 +113,16 @@
   }
 }
 
+.no-outline:focus,
+.no-outline:hover {
+  outline: 1px solid rgb(255, 255, 255);
+}
+
+.has-outline:focus,
+.has-outline:hover {
+  outline: 1px solid rgb(255, 255, 255);
+}
```

Comment:
Hmm... @kivuvarosekivuvan based on the findings you wrote in our slack chat (and now that I think about it more), we don't need a new class-name in the css. We also don't need all those javascript code. You can happily remove all the code you have added.

I've also seen your comment here on using focus-within inside of the company info. I think it may be more helpful and clear to developers if we target the class from the library/framework directly.

I'll suggest a more straightforward fix using only css. At the bottom of this `company-info.component.scss` file, you will find the `.mat-expansion-panel` class.

**_Step 1:_** You can restore the **_focus_** of the `.mat-expansion-pane-header` from within here.

Now, after doing this, most parts of the header should be hidden. This is because `mat-expansion-panel` element itself has some hidden overflow applied to it.

**_Step 2:_** You can also just restore the overflow of the `mat-expansion-panel` element from within there also.

Really, that's all.